spec/features/welcome_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe "Welcome Page" do
```

Comment:
Nice robust test assertions here! If this were a project I would worry about how delicate these tests are given that you're using VCR and the data could change (ie. they could add a food result and that number could go up to 49652) but for an assessment this is a great level of robustness